kore/src/main/java/org/kframework/utils/errorsystem/KEMException.java

```diff
@@ -125,7 +125,7 @@ public String getMessage() {
         return exception.toString();
     }
 
-    private static KEMException create(ExceptionType type, KExceptionGroup group, String message,
+    public static KEMException create(ExceptionType type, KExceptionGroup group, String message,
```

Comment:
this was private for a reason. why did you make it public?

Reply:
I need to recreate a copy of KExceptions thrown by the parser but with the Source and Location replaced.
I had to expose `exceptionGroup` through a getter and make the general constructor public.
Alternatively, if you want, I could make a function `KMException.updateLocation`. But that seemed custom to me.